docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -4,48 +4,56 @@ description: List of operations that users can perform on Codacy depending on th
 
 # Roles and permissions for organizations
 
-Your team members have different permission levels on Codacy depending on their role on your Git provider. To change the permission level of a user on Codacy, you must adjust their role directly on your Git provider so that Codacy will use the corresponding permission level on the next time that the user logs in to Codacy.
+By default, Codacy assigns each organization member a role corresponding to that member's role on your Git provider.
 
-See the Codacy permission levels that correspond to each role on your Git provider:
+To update a member's role on Codacy, update that member's role directly on your Git provider. When next logging in to Codacy, the member is assigned the new role.
+
+To review the permissions granted by each role, see the tables for each Git provider:
 
 -   [GitHub](#permissions-for-github)
 -   [GitLab](#permissions-for-gitlab)
 -   [Bitbucket](#permissions-for-bitbucket)
 
-See [managing people](managing-people.md) to list and manage the members of your Codacy organization.
+!!! tip
+    You can grant some administrative permissions to any organization member using the [Organization Manager](#the-organization-manager-role) role.
+
+To list and manage the members of your Codacy organization, see the [Managing people](managing-people.md) page.
 
 ## Permissions for GitHub
 
-The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresponding Codacy permission levels and the operations that they're allowed to perform:
+The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresponding Codacy roles and the operations that they're allowed to perform:
 
 <table>
   <thead>
     <tr>
       <th>GitHub role</th>
-      <th>Outside<br/>Collaborator<sup>1</sup></th>
-      <th>Repository<br/>Read</th>
-      <th>Repository<br/>Triage</th>
-      <th>Repository<br/>Write</th>
-      <th>Repository<br/>Maintain</th>
-      <th>Repository<br/>Admin</th>
+      <th>Outside<br/>collaborator<sup>1</sup></th>
+      <th>Repository<br/>read</th>
+      <th>Repository<br/>triage</th>
+      <th>Repository<br/>write</th>
+      <th>Repository<br/>maintain</th>
+      <th>Repository<br/>admin</th>
+      <th>-</th>
       <th>Organization<br/>Owner</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td>Codacy permission level</td>
+      <td>Codacy role</td>
       <td>-</td>
-      <td colspan="2">Repository<br/>Read</td>
-      <td colspan="2">Repository<br/>Write</td>
-      <td>Repository<br/>Admin</td>
-      <td>Organization<br/>Admin</td>
+      <td colspan="2">Repository<br/>read</td>
+      <td colspan="2">Repository<br/>write</td>
+      <td>Repository<br/>admin</td>
+      <td><a href="#the-organization-manager-role">Organization<br/>manager</a></td>
```

Comment:
It depends. This order matches the "power" of each role, with the organization mdmin, at the very end of the table, being the one with most permissions. Visually, placing the organization manager at the end may signal that it can do more than the org admin, which is not the case. I'd leave it like this for now and review when we add a new role soon, the billing manager.

Reply:
Fair enough! Let's revisit these tables when we have the billing manager role. 👍